plugin/action/throttle/throttle.go

```diff
@@ -83,6 +127,11 @@ type Config struct {
 	//> * `limit_kind` – the type of a limit: `count` - number of messages, `size` - total size from all messages
 	//> * `conditions` – the map of `event field name => event field value`. The conditions are checked using `AND` operator.
 	Rules []RuleConfig `json:"rules" default:"" slice:"true"` //*
+
+	//> @3@4@5@6
+	//>
+	//> IsThrottleDistributed regulates global tracing for service.
+	IsThrottleDistributed bool `json:"is_throttle_distributed" default:"false"` //*
```

Comment:
why you need this parameter?

Reply:
don't need to duplicate parameter name in the comment

Reply:
Param were removed due redundancy